tox.ini

```diff
@@ -37,3 +31,31 @@ commands =
     flake8 src setup.py
     check-manifest
     check-python-versions
+
+[testenv:coverage]
+basepython = python3
+setenv =
+    COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+deps =
+    coverage
+    coverage-python-version
+    zope.testrunner
+commands =
+    coverage erase
+    coverage run -m zope.testrunner --test-path=src []
+    coverage combine
+    coverage html
+    coverage report -m --fail-under=42
+
+[coverage:report]
+precision = 2
+exclude_lines =
+    pragma: nocover
+    except ImportError:
+    raise NotImplementedError
+    if __name__ == '__main__':
+    self.fail
+    raise AssertionError
+
+[coverage:html]
+directory = htmlcov
```

Comment:
You're splitting coverage's configuration across two files: `.coveragerc` and here.  I don't like that.  I'm not sure coverage.py even supports this.

Reply:
I fixed it, it is not supported and it was a mistake on my side.